interfaces/INetflix.h

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Shouldn't those changes be in the IApplication.h instead of INetflix.h?

Reply:
Please do not mix tabs with space. Use consistent indentation (spaces)

Reply:
Are we going to leave any methods inside the INetflix interface?

Reply:
too fast, too furious, sorry